src/scripts.js

```diff
@@ -92,20 +92,17 @@ function welcomeUser() {
 
 }
 
-// function convertFriendIdToName() {
-//   const friendsArray = []
-//         singleUser.friends.forEach(friend => {
-//           console.log(friend)
-//           const matchingIds = users.find(user => {
-//             user.id === friend;
-//           })
-//          matchingIds.forEach(user =>  friendsArray.push(user.name))
-//
-//         })
-//         console.log('array', friendsArray)
-//         return friendsArray
-//
-//   }
+function convertFriendIdToName() {
+  let friendArray = []
+  const friendIds = users.filter(user => {
+    if(singleUser.friends.includes(user.id)) {
```

Comment:
I like the logic here. I like how you used filter and forEach (that's similar to the thinking I had) and I noticed how you used the entire users array to iterate over and then referred to singleUser!